src/Struct/V2/Order/PaymentSource/Common/ExperienceContext.php

```diff
@@ -70,6 +71,9 @@ class ExperienceContext extends Struct
     #[OA\Property(type: 'array', items: new OA\Items(type: 'string'))]
     protected array $customerServiceInstructions;
 
+    #[OA\Property(ref: '#/components/schemas/paypal_v2_order_payment_source_common_attributes_order_update_callback_config')]
```

Comment:
Use the class string instead, it will be resolved correctly on generation:
```suggestion
    #[OA\Property(ref: OrderUpdateCallbackConfig::class)]
```